<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_widget</class>
 <widget class="QWidget" name="search_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>search_widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/app.qrc">
    <normaloff>:/icon/icon/搜索.ico</normaloff>:/icon/icon/搜索.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="about_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/app.qrc">
         <normaloff>:/icon/icon/关于.ico</normaloff>:/icon/icon/关于.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="json_file_path_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>题库.json</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open_json_file_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>打开json文件</string>
       </property>
       <property name="icon">
        <iconset resource="../res/app.qrc">
         <normaloff>:/icon/icon/json.ico</normaloff>:/icon/icon/json.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copy_simple_json_pushButton">
       <property name="text">
        <string>复制示例json数据</string>
       </property>
       <property name="icon">
        <iconset resource="../res/app.qrc">
         <normaloff>:/icon/icon/复制.ico</normaloff>:/icon/icon/复制.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="select_search_book_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>肖秀荣1000题</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>腿姐300天70分</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search_keyword_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>选项</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>搜索关键字</string>
       </property>
       <property name="icon">
        <iconset resource="../res/app.qrc">
         <normaloff>:/icon/icon/搜索.ico</normaloff>:/icon/icon/搜索.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="search_nums_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>3条记录</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="res_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="maxVisibleItems">
        <number>500</number>
       </property>
       <item>
        <property name="text">
         <string>0-...曾接受过那个时代的唯心主义和资...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1-...曾接受过那个时代的唯心主义和资...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2-...曾接受过那个时代的唯心主义和资...</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pre_res_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/app.qrc">
         <normaloff>:/icon/icon/上一首-01.ico</normaloff>:/icon/icon/上一首-01.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_res_pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/app.qrc">
         <normaloff>:/icon/icon/下一首-01.ico</normaloff>:/icon/icon/下一首-01.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="question_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>题目-单选</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="local_chapter_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>所在章节：第一章：马克思主义是关于无产阶级和人类解放的科学</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="question_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>马克思和恩格斯并不是先知先觉的圣人，他们从小面对的也是一个充满矛盾的现实世界，在家庭、学校和社会的影响下，也曾接受过那个时代的唯心主义和资产阶级民主主义思想。但是资本主义制度的弊端和劳动群众渴求解放的呼声，促使他们立志进行社会变革。并走上求索科学真理的道路。表明马克思和恩格斯完成了从唯心主义向唯物主义、从革命民主主义向共产主义的转变的是</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="A_checkBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>A.世界上第一个无产阶级政党的党纲《共产党宣言》的发表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="B_checkBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>B.马克思的《哥达纲领批判》和恩格斯的《反杜林论》</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="C_checkBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>C.马克思和恩格斯发表在《德法年鉴》上的论文</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="D_checkBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>D.马克思和恩格斯在巴黎合写的《神圣家族》一书</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ans_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>答案</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QPushButton" name="show_ans_pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>显示答案</string>
            </property>
            <property name="icon">
             <iconset resource="../res/app.qrc">
              <normaloff>:/icon/icon/参考答案.ico</normaloff>:/icon/icon/参考答案.ico</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="is_auto_show_ans_checkBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>自动显示答案</string>
            </property>
            <property name="icon">
             <iconset resource="../res/app.qrc">
              <normaloff>:/icon/icon/自动化.ico</normaloff>:/icon/icon/自动化.ico</iconset>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="ans_label">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>正确选项：C</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="copy_this_question_json_pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>例如：
{
    &quot;书本&quot;: &quot;xxx&quot;,
    &quot;章节&quot;: &quot;xxx&quot;,
    &quot;题目&quot;: &quot;xxx&quot;,
    &quot;选项&quot;: [
        &quot;xxx&quot;,
        &quot;xxx&quot;,
        &quot;xxx&quot;,
        &quot;xxx&quot;
    ],
    &quot;答案&quot;: [
        0,
        1,
        2
    ],
    &quot;解析&quot;: &quot;xxx&quot;
}</string>
            </property>
            <property name="text">
             <string>复制(json数据)</string>
            </property>
            <property name="icon">
             <iconset resource="../res/app.qrc">
              <normaloff>:/icon/icon/复制.ico</normaloff>:/icon/icon/复制.ico</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="copy_this_question_format_pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>例如：
题目：xxx
A.xxx
B.xxx
C.xxx
D.xxx

参考答案：xxx
解析：xxxxxx</string>
            </property>
            <property name="text">
             <string>复制(格式化后的)</string>
            </property>
            <property name="icon">
             <iconset resource="../res/app.qrc">
              <normaloff>:/icon/icon/复制.ico</normaloff>:/icon/icon/复制.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="analysis_label">
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>马克思和恩格斯发表在1844年2月《德法年鉴》上的论文表明,他们完成了从唯心主义向唯物主义、从革命民主主义向共产主义的转变,为创立马克思主义奠定了思想前提。C正确。1844年8月底,马克思和恩格斯在巴黎会面。此前他们曾有过一面之缘,但这次会面使他们发现彼此的基本观点完全一致,从此开始了毕生的合作。他们先是在巴黎合写了《神圣家族》一书,接着又在布鲁塞尔合写了《德意志意识形态》。1848年2月,《共产党宣言》发表,标志着马克思主义的公开问世。1871年3月,巴黎工人起义并成立巴黎公社,马克思代表第一国际写出了著名的《法兰西内战》,高度赞扬了巴黎工人的伟大创举,科学总结了巴黎公社的历史经验。该书与几年后他写的《哥达纲领批判》一起,进一步丰富了科学社会主义学说。1876一1878年。恩格斯写出了《反杜林论》,全面阐述了马克思主义理论体系。A、B、D不符题意。</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/app.qrc"/>
 </resources>
 <connections/>
</ui>
